Colours_And_Fonts_style_v1.css

```diff
@@ -128,31 +194,55 @@ ul li:hover ul {
 
 .box1 {
   width: 1990px;
-  height: 20px;
-  background-color: #BC1327;
+  height: 10px;
+  background-color: #A2091B;

```

Comment:
I changed the height from 20px to 10px as 20px was to big so I made it smaller. I also changed the colour from a light red to a darker red to match the Tawa College colours.